man/systemd.syntax.xml

```diff
@@ -95,6 +95,10 @@ KeyTwo=value 2 \
     <option>false</option> and <option>off</option> are
     equivalent.</para>
 
+    <para>Comprehensive boolean arguments take the same format as boolean
+    arguments, as well as <option>always</option> or
+    <option>never</option>.</para>
```

Comment:
let's not add this here unless at least two unrelated parts use them. If initially we only use that for the cgroup stuff, let's document it there